src/main/java/digital/slovensko/autogram/core/AppStarter.java

```diff
@@ -20,7 +20,8 @@ public class AppStarter {
         addOption("f", "force", false, "Overwrite existing file(s).").
         addOption(null, "pdfa", false, "Check PDF/A compliance before signing.").
         addOption(null, "parents", false, "Create all parent directories for target if needed.").
-        addOption("d", "driver", true, "PCKS driver name for signing. Supported values: eid, secure_store, monet, gemalto.").
+        addOption("d", "driver", true, "PCKS driver name for signing. Supported values: eid, secure_store, monet, gemalto, keystore.").
+        addOption(null, "keystore", true, "Absolute path to a keystore file that can be used for signing.").
```

Comment:
Tu netreba ten option na heslo?

Reply:
V cli sa dá "napísať" prázdne heslo - iba enter bez ničoho. To sa ale v gui nedá, lebo tam máme validáciu na tom inpute na heslo. Možno stojí zazváženie, či nevyhodiť tú validáciu na gui potom.